neptune/new/internal/init_impl.py

```diff
@@ -326,7 +329,8 @@ def init(project: Optional[str] = None,
             run_id=api_run.short_id
         ))
 
-    uncaught_exception_handler.activate()
+    if fail_on_exception:
```

Comment:
Is this `if` necessary? In future we can have many handlers registered.

Reply:
No, I should remobe this